static/monorail-2.0.yaml

```diff
@@ -650,6 +665,162 @@ paths:
+          schema:
+            $ref: '#/definitions/Error'
+      summary: Post a hook
+      tags:
+      - /api/2.0
+      x-authentication-type:
+      - jwt
+      x-privileges:
+      - Write
+      - hooksWrite
+      x-swagger-schema: hooks.2.0.json#/definitions/HookBase
+      x-view: hook.2.0.json
+    x-swagger-router-controller: hooks
+  /hooks/{identifier}:
+    get:
+      description: Get hook by id
+      operationId: hooksGetById
+      parameters:
+      - description: id of the hook you wish to delete
+        in: path
+        name: identifier
+        required: true
+        type: string
+      responses:
+        200:
+          description: Successfully retrieved the specified hook
+          schema:
+            type: object
+        default:
+          description: Unexpected error
+          schema:
+            $ref: '#/definitions/Error'
+      summary: Get all hooks
+      tags:
+      - /api/2.0
+      x-authentication-type:
+      - jwt
+      x-privileges:
+      - Read
+      - hooksRead
+    delete:
+      description: Delete a hook based on hook id.
+      operationId: hooksDelById 
+      parameters:
+      - description: id of the hook you wish to delete
+        in: path
+        name: identifier
+        required: true
+        type: string
+      responses:
+        204:
+          description: Hook successfully deleted
+        404:
+          description: Hook not found.
+          schema:
+            $ref: '#/definitions/Error'
+        500:
+          description: Error deleting hook from the database
+          schema:
+            $ref: '#/definitions/Error'
+        default:
+          description: Unexpected error
+          schema:
+            $ref: '#/definitions/Error'
+      summary: Delete a hook
+      tags:
+      - /api/2.0
+      x-authentication-type:
+      - jwt
+      x-privileges:
+      - Remove
+      - hooksRemove
+    patch:
+      description: Modify the properties of a hook.
+      operationId: hooksPatchById
+      parameters:
+      - description: The id of the hook to patch
+        in: path
+        name: identifier
+        required: true
+        type: string
+      - description: The hook properties to be modified
+        in: body
+        name: body
+        required: true
+        schema:
+          $ref: '#/definitions/hooks.2.0_HooksBase'
+      responses:
+        200:
+          description: Successfully modified the hook
+          schema:
+            type: object
+        404:
+          description: The specified hook was not found
+          schema:
+            $ref: '#/definitions/Error'
+        default:
+          description: Unexpected error
+          schema:
+            $ref: '#/definitions/Error'
+      summary: Patch a hook
+      tags:
+        - /api/2.0
+      x-authentication-type:
+        - jwt
+      x-privileges:
+        - Write
+        - hooksModify
+      x-swagger-schema: hooks.2.0.json#/definitions/HookBase
```

Comment:
@pengz1 Thanks!  I think you missed one reference to `HookPost` (see my other comment), but looks good otherwise.